BitwardenShared/Core/Auth/Services/API/Auth/Requests/IdentityTokenRequest.swift

```diff
@@ -80,9 +80,7 @@ struct IdentityTokenRequest: Request {
                 from: response.body
             ) else { return }
 
-            if let twoFactorProviders = errorModel.twoFactorProviders,
-               var twoFactorProvidersData = errorModel.twoFactorProvidersData {
-                twoFactorProvidersData.providersAvailable = twoFactorProviders
+            if let twoFactorProvidersData = errorModel.twoFactorProvidersData {
```

Comment:
🤔 I think we should also be checking `let errorModel.twoFactorProvidersData.providersAvailable` to actually check there are 2FA providers available in the response data. Because if the response sends an empty 2FA providers data, then the current code won't fail and it should.